mmdet/models/dense_heads/anchor_head.py

```diff
@@ -510,6 +511,7 @@ def get_bboxes(self,
                 level with shape (N, num_anchors * 4, H, W)
             img_metas (list[dict]): Meta information of each image, e.g.,
                 image size, scaling factor, etc.
+            imgs (list[torch.Tensor]): List of multiple images
```

Comment:
If you only want to use input image shape, better to get the shape as tensor at preprocessing step and feed it to img_metas. Then, there's no change to the interface of `get_bboxes` method.
For example :
```
input_shape_t = one_metas[img_id]['input_shape_t']
```

Reply:
I tried this, but it is still a constant in the model, like this:
![image](https://user-images.githubusercontent.com/27288350/109442972-612bfd80-7a74-11eb-9145-c68cec182f1a.png)
We should turn it into a calculation related to the input shape，like this:
![image](https://user-images.githubusercontent.com/27288350/109443247-08109980-7a75-11eb-9b42-8eff0879a92e.png)

Reply:
That's a bit strange. I'll try it as well.